monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinitionModel.java

```diff
@@ -24,6 +24,9 @@
 
 public class ScheduleDefinitionModel {
     private final String _hotFolder;
+    
+    @JsonProperty("exportMetrics")
+    private boolean _exportMetrics;
 
     @JsonCreator
     public ScheduleDefinitionModel(@JsonProperty("hotFolder") final String hotFolder) {
```

Comment:
I was considering the immutability too, but I'm afraid we may want to add more options later and that will result in a constructor with a lot of arguments. Just say it, if you'd rather have it immutable, I'll change it.